.github/workflows/check_plugin.yml

```diff
@@ -1,15 +1,18 @@
-name: Check Plugin All Availability
+name: Check Plugin Availability
 
 on:
-  # schedule:
-  #   - cron: '30 9 * * *'
+  schedule:
+    - cron: '30 9 * * *'
```

Comment:
Let's change the schedule. 9AM or 6PM is better with PST timezone since the chime group is used for notification

Reply:
Resolve by setting to 30 1/16 * * *, which is 6:30pm and 9:30am PST.